pkg/storage/kubernetes/client.go

```diff
@@ -95,8 +92,10 @@ func (i *Client) ListIPPools(ctx context.Context) ([]storage.IPPool, error) {
 	return whereaboutsApiIPPoolList, nil
 }
 
-func (i *Client) ListPods(ctx context.Context) ([]v1.Pod, error) {
-	ctxWithTimeout, cancel := context.WithTimeout(ctx, i.timeout)
+func (i *Client) ListPods() ([]v1.Pod, error) {
+	logging.Debugf("listing Pods")
+
+	ctxWithTimeout, cancel := context.WithTimeout(context.Background(), listRequestTimeout)
```

Comment:
Cool, I think I get it, you've got all the timeouts normalized on the listRequestTimeout and then we can eliminate the other timeouts in the reconciler. Nicely done.

Reply:
That's correct. I will add a description in the commit. Thanks!